packages/hdwallet-phantom/src/phantom.ts

```diff
@@ -283,21 +315,22 @@ export class PhantomHDWallet extends PhantomHDWalletInfo implements core.HDWalle
 
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   public async ethSignTx(msg: core.ETHSignTx): Promise<core.ETHSignedTx | null> {
-    throw new Error("Method not implemented");
+    console.error("Method not implemented");
+    return null;
   }
 
   public async ethSendTx(msg: core.ETHSignTx): Promise<core.ETHTxHash | null> {
-    const address = await this.ethGetAddress();
+    const address = await this.ethGetAddress({ addressNList: msg.addressNList });
     return address ? eth.ethSendTx(msg, this.evmProvider, address) : null;
   }
 
   public async ethSignMessage(msg: core.ETHSignMessage): Promise<core.ETHSignedMessage | null> {
-    const address = await this.ethGetAddress();
+    const address = await this.ethGetAddress({ addressNList: msg.addressNList });
```

Comment:
I added it to respect the hdwallet-core interface. I can add a throw if not account 0 as an extra rule if we want, otherwise it is just passing through like metamask